api/tests/phones_views_tests.py

```diff
@@ -1026,6 +1026,26 @@ def test_inbound_call_valid_twilio_signature_disabled_number(
     mocked_twilio_client.messages.create.assert_not_called()
 
 
+def test_inbound_call_valid_twilio_signature_no_remaining_seconds(
```

Comment:
It seems weird that the `test_voice_status_completed_reduces_remaining_seconds` test didn't catch this error? Should we update that test to make sure it's checking the right attribute is decremented?

Reply:
Added comments on a9314a8 why this test didn't catch the error

Reply:
Cool, thanks for the `TODO`